core/block_svg.js

```diff
@@ -1664,9 +1664,19 @@ Blockly.BlockSvg.prototype.render = function(opt_bubble) {
   }
   Blockly.utils.dom.stopTextWidthCache();
 
-  var cursor = this.workspace.getCursor();
-  if (this.workspace.keyboardAccessibilityMode && this.pathObject.cursorSvg_) {
-    cursor.draw();
+  this.updateAccessibility_();
+};
+
+/**
+ * Redraw any attached marker or cursor svgs if needed.
+ * @protected
+ */
+Blockly.BlockSvg.prototype.updateAccessibility_ = function() {
```

Comment:
I would rather we use updateMarkers_ instead of updateAccessibility. Markers can be used outside of accessibility purposes and I don't feel like the name is very descriptive of what is actually happening.

Reply:
Done.